api/logs/views.py

```diff
@@ -220,7 +221,7 @@ class NodeLogContributors(JSONAPIBaseView, generics.ListAPIView, ODMFilterMixin,
     required_read_scopes = [CoreScopes.USERS_READ]
     required_write_scopes = [CoreScopes.NULL]
 
-    serializer_class = UserSerializer
+    serializer_class = UnregisteredContributorSerializer
```

Comment:
Another option might be to add the `unregistered_contributor` to the `UserSerializer`, and let that serializer figure out the appropriate representation for that user (claimed or unclaimed).  Then 
```
 +        modified_users = []
 +        for contrib in associated_users:
 +            if contrib.unclaimed_records.get(log.original_node._id):
 +                contrib.unregistered_name = contrib.unclaimed_records[log.original_node._id]['name']
 +            modified_users.append(contrib)
 +        return modified_users
```
Could go away. Really what this ^  does is tamper with serialization in the view. which is not ideal.